tests/main/mantine_test.cljs

```diff
@@ -19,20 +25,22 @@
      ($ NavLink {:label "b"
                  :href "www.example.com/b"})))
 
-(deftest mantine-test
-  (testing "should render mantine component links"
-    (async done
-      (p/catch
-        (p/let [groups (tlr/waitFor #(-> (tlr/render
-                                          ($ MantineProvider ($ MyNavLinks)))
-                                         (.findByTestId "link-groups")))
-                links (->> (.querySelectorAll groups ".mantine-NavLink-root")
-                           (mapv #(-> % .-href (str/split "/") last)))]
+(defnc MyCounter []
```

Comment:
I think we can add this to the component.cljs file, because I wrote the same thing to test the portifolio lib, we could move and use in both (test/portifolio) the same sample component.

Reply:
Makes sense! I updated the PR, could you check if it addresses it?